string object_name
---
Detection[] detections
bool ok